src/sagas.js

```diff
@@ -354,7 +354,12 @@ export function* fetchSingleVm (action) {
       internalVm.sessions = yield fetchVmSessions({ vmId: internalVm.id })
       // TODO: Support <4.2 for snapshots?
     }
-
+    for (let i in internalVm.snapshots) {
```

Comment:
Do you plan on checking other `follow` options or opening a BZ on the rest api to get that functionality (potentially in the future)?